src/cli.rs

```diff
@@ -11,6 +11,7 @@ use std::path::PathBuf;
 pub struct Cli {
     #[command(subcommand)]
     pub command: Option<Commands>,
+    #[cfg(not(feature = "local-only"))]
```

Comment:
Perhaps we should not kill the ability to use tpi to control another. I would still change the default to '127.0.0.1'